x/gov/context.go

```diff
@@ -0,0 +1,56 @@
+package gov
+
+import (
+	"context"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/x"
+)
+
+type contextKey int // local to the gov module
```

Comment:
No need for this comment. It is lower case and compiler assures that this is local to this package.